src/commands/sync.ts

```diff
@@ -126,6 +126,13 @@ export const commandSync = command({
     await uploadBigFiles(state, bigFiles, target);
     // Tar small files and upload them
     await uploadSmallFiles(state, smallFiles, target);
+
+    // Force a scan after the upload completes
+    if (state.scan === false) {
+      state.scan = true;
+      await uploadBigFiles(state, bigFiles, target);
```

Comment:
What will happen if the scan failure and found some upload failure files?

Reply:
After a upload completes, this will scan the target folder and attempt to upload any missing files.